test/e2e/prometheus-metrics-test

```diff
@@ -139,9 +140,9 @@ fi
 POD_NAME=$(get_nth_worker_pod)
 echo "✅ Fetched the pod $POD_NAME "
 
-kubectl -n kube-system port-forward $POD_NAME 7000:9092 &
+kubectl -n kube-system port-forward "$POD_NAME" 7000:9092 &
 PORT_FORWARD_PID=$!
-trap "kill ${PORT_FORWARD_PID}" EXIT SIGINT SIGTERM ERR
+trap 'kill ${PORT_FORWARD_PID}' EXIT SIGINT SIGTERM ERR
```

Comment:
just curious, do you remember why it recommends single quote here?

Reply:
it's to make sure the expansion happens on the trap signal rather than at definition. In this case, it doesn't matter, so I went with the recommendation rather than the shellcheck disable line above.